config/routes.rb

```diff
@@ -1,33 +1,13 @@
 Rails.application.routes.draw do
 
-  root 'static_pages#home'
 
```

Comment:
@HacksForHugs you need `root 'static_pages#home`. i think you may have deleted too much from your merge conflicts....